lib/connectors/mongodb/connector.rb

```diff
@@ -153,10 +153,10 @@ def create_db_cursor_on_collection(collection)
       def create_aggregate_cursor(collection)
         aggregate = @advanced_filter_config[:aggregate]
 
-        pipeline = aggregate[:pipeline]
+        pipeline = aggregate[:pipeline] || []
         options = extract_options(aggregate)
 
-        if !pipeline.nil? && pipeline.empty? && !options.present?
+        if !pipeline.present? && !options.present?
```

Comment:
Makes totally sense, adjusted it 👍 What's the rationale behind stopping to use ActiveSupport?

Reply:
> What's the rationale behind stopping to use ActiveSupport?

ActiveSupport is a part of RubyOnRails framework where performance and clean code are not super important (not saying it's bad).  A recent example was HashWithIndifferentAccess which adds unnecessary overhead.

I don't like extending the language without any reasonable need because this kind of libs often bring overhead.